mkosi/__init__.py

```diff
@@ -6434,8 +6463,10 @@ def print_summary(args: CommandLineArguments) -> None:
         f"    Output nspawn Settings: {none_to_na(args.output_nspawn_settings if args.nspawn_settings is not None else None)}"
     )
     MkosiPrinter.info(
-        f"                   SSH key: {none_to_na((args.ssh_key or args.output_sshkey) if args.ssh else None)}"
+        f"                   SSH key: {none_to_na((args.ssh_key or args.output_sshkey or args.ssh_agent) if args.ssh else None)}"
```

Comment:
This is actually the reason why I'm saying this, since a value of `False` wouldn't really mesh with `none_to_na`, but a value of `None` would.